lib/ansible/modules/cloud/docker/docker_node.py

```diff
@@ -122,7 +122,7 @@
 '''
 
 RETURN = '''
-node_facts:
+node:
```

Comment:
Why not `node_info`?

Reply:
For consistency: `docker_node_facts` returns `nodes`, and most other modules which operate on one object return a variable whose name is the object type in singular without some suffix.